pennylane/math/matrix_manipulation.py

```diff
@@ -13,28 +13,29 @@
 # limitations under the License.
 """This module contains methods to expand the matrix representation of an operator
 to a higher hilbert space with re-ordered wires."""
-import copy
+import itertools
 from functools import reduce
 from typing import Generator, Iterable, Tuple
 
 import numpy as np
-from scipy.sparse import csr_matrix, eye, issparse, kron
+from scipy.sparse import csr_matrix, eye, kron
 
 import pennylane as qml
 from pennylane.wires import Wires
 
 
-def expand_matrix(base_matrix, wires, wire_order=None, sparse_format="csr"):
-    """Re-express a base matrix acting on a subspace defined by a set of wire labels
+def expand_matrix(mat, wires, wire_order=None, sparse_format="csr"):
+    # pylint: disable=too-many-branches
+    """Re-express a matrix acting on a subspace defined by a set of wire labels
     according to a global wire order.
 
     Args:
-        base_matrix (tensor_like): base matrix to expand
-        wires (Iterable): wires determining the subspace that base matrix acts on; a base matrix of
+        mat (tensor_like): matrix to expand
+        wires (Iterable): wires determining the subspace that `mat` acts on; a matrix of
```

Comment:
```suggestion
        wires (Iterable): wires determining the subspace that ``mat`` acts on; a matrix of
```
Sphinx needs double back-ticks.